source/utils/format.spec.ts

```diff
@@ -55,4 +56,19 @@ describe('Format', () => {
     expect(input.length).toBeGreaterThanOrEqual(output.length);
     expect(output.endsWith('...')).toBe(true);
   });
+
+  //* formatSeedPhrase
+  // remove double spaces, numbers and symbols
+  // transform in lowerCase
+  it('should format a Seed Phrase', () => {
+    const input =
+      ' peace? uncle! Grit   essence3 stuff angle Cruise annual fury letter snack globe ';
```

Comment:
if there is a number or special character we can't try to guess the missing letter in the word, in this case we should only throw an error

we can only format if there is no number/special character